src/components/UI/downloads/DownloadsTable.tsx

```diff
@@ -37,6 +37,15 @@ export const DownloadsTable: FC<Props> = ({
 
   const LAST_2_LINUX_RELEASES = amountOfReleasesToShow + 12;
 
+  const getDefaultIndex = useMemo<number>(() => {
+    const OS: string = typeof window !== 'undefined' ? window.navigator.platform : '';
```

Comment:
I think this check doesn't work on mobile browsers

Reply:
Gotcha... okay, it seems to be working on iOS, but I'll look into it